Pod/Classes/VideoComponent/ARVideoManager.m

```diff
@@ -0,0 +1,48 @@
+#import <AVFoundation/AVFoundation.h>
+#import "ARVideoManager.h"
+
+
+@interface ARVideo : UIView
+@property (nonatomic, strong, readwrite) AVPlayerLayer *playerLayer;
+@end
+
+
+@implementation ARVideo
+
+- (void)setFrame:(CGRect)frame
+{
+    [super setFrame:frame];
+    self.playerLayer.frame = self.bounds;
+}
+
+@end
+
+
+@implementation ARVideoManager
+RCT_CUSTOM_VIEW_PROPERTY(source, NSDictionary, ARVideo)
+{
+    NSDictionary *source = [RCTConvert NSDictionary:json];
+    id uri = [source objectForKey:@"uri"];
```

Comment:
Modeled this after (what seems to be) RN standard practices -- `source: { uri: '...' }`.

Reply:
Yeah, FB didn't like image src being a string - so this fits 👍